test/sqlancer/reducer/TestEnvironment.java

```diff
@@ -21,8 +21,26 @@ public class TestEnvironment {
     private VirtualDBProvider provider = null;
     private VirtualDBGlobalState state, newGlobalState;
 
-    public TestEnvironment() throws Exception {
+    private Reducer<VirtualDBGlobalState> reducer = null;
+
+    private static final int USING_STATEMENT_REDUCER = 1;
```

Comment:
Here, we should use a Java enum, but it's also fine to do this in a follow-up PR.